spec/mollie/client_spec.rb

```diff
@@ -62,6 +62,42 @@
     end
   end
 
+  context 'methods' do
+    it 'returns a hash with methods' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.